### PR TITLE
update hardfork table with height and version info

### PR DESCRIPTION
**Pull request by Gingeropolous**

### CODE REVIEW

**Comment by Gingeropolous** [comment]
ugh. i don't wanna search for them in the code :/

**Comment by hyc** [comment]
@Gingeropolous all here https://github.com/monero-project/monero/blob/master/src/cryptonote_core/blockchain.cpp#L93

File: README.md [hunk 1]
```diff
@@ -95,12 +95,12 @@ Monero uses a fixed-schedule hard fork mechanism to implement new features. This
...
 
 
-| Fork Date              | Consensus version | Minimum Monero Version | Recommended Monero Version | Details            |  
+| Block Height / Fork Date              | Consensus version | Minimum Monero Version | Recommended Monero Version | Details            |  
 | ----------------- | ----------------- | ---------------------- | -------------------------- | ------------------ |
 | 2016-09-21        | v3                | v0.9.4                 | v0.10.0                    | Splits coinbase into denominations  |
 | 2017-01-05        | v4                | v0.10.1                 | v0.10.2.1                   | Allow normal and RingCT transactions |
 | 2017-04-15        | v5                | v0.10.3.0               | v0.10.3.1                    | Adjusted minimum blocksize and fee algorithm      |
-| 2017-09-21        | v6                | Not determined as of 2017-03-27                | Not determined as of 2017-03-27                    | Allow only RingCT transactions      |
+| 1400000 / 2017-09-16        | v6                | v0.11.0.0                | v0.11.0.0                    | Allow only RingCT transactions, allow only >= ringsize 5      |
```
Comment (NanoAkron) [inline README.md]: 'Allow only RingCT transactions, and ringsize >= 5'?

Comment (MaxXor) [reply README.md]: @NanoAkron The `,` is not needed. Other than this I prefer yours.

File: README.md [hunk 2]
```diff
@@ -95,12 +95,13 @@ Monero uses a fixed-schedule hard fork mechanism to implement new features. This
...
+| Block Height | Fork Date              | Consensus version | Minimum Monero Version | Recommended Monero Version | Details            |  
+| ----------------- | ----------------- | ---------------------- | -------------------------- | ------------------ | ---------- |
+| 1009827 | 2016-04-20 	      | v2                | v0.9.4                 | v0.9.4                     |  Allow only >= ringsize 3, blocktime = 120 seconds       |
+| 1141317 | 2016-09-21        | v3                | v0.9.4                 | v0.10.0                    | Splits coinbase into denominations  |
+| 1220516 | 2017-01-05        | v4                | v0.10.1                 | v0.10.2.1                   | Allow normal and RingCT transactions |
+| 1288616 | 2017-04-15        | v5                | v0.10.3.0               | v0.10.3.1                    | Adjusted minimum blocksize and fee algorithm      |
+| 1400000 | 2017-09-16        | v6                | v0.11.0.0                | v0.11.0.0                    | Allow only RingCT transactions, allow only >= ringsize 5      |
```
Comment (MaxXor) [inline README.md]: Please change it to:
> Allow only RingCT transactions and ringsize >= 5

File: README.md [hunk 3]
```diff
@@ -95,12 +95,13 @@ Monero uses a fixed-schedule hard fork mechanism to implement new features. This
...
-| ----------------- | ----------------- | ---------------------- | -------------------------- | ------------------ |
-| 2016-09-21        | v3                | v0.9.4                 | v0.10.0                    | Splits coinbase into denominations  |
-| 2017-01-05        | v4                | v0.10.1                 | v0.10.2.1                   | Allow normal and RingCT transactions |
-| 2017-04-15        | v5                | v0.10.3.0               | v0.10.3.1                    | Adjusted minimum blocksize and fee algorithm      |
-| 2017-09-21        | v6                | Not determined as of 2017-03-27                | Not determined as of 2017-03-27                    | Allow only RingCT transactions      |
+| Block Height | Fork Date              | Consensus version | Minimum Monero Version | Recommended Monero Version | Details            |  
+| ----------------- | ----------------- | ---------------------- | -------------------------- | ------------------ | ---------- |
+| 1009827 | 2016-04-20 	      | v2                | v0.9.4                 | v0.9.4                     |  Allow only >= ringsize 3, blocktime = 120 seconds       |
```
Comment (MaxXor) [inline README.md]: Please change to:
> Allow only ringsize >= 3 and adjusted blocktime to 120 seconds

**Comment by Gingeropolous** [comment]
apparently the repo maintainer can squash on github now, https://github.com/blog/2141-squash-your-commits ... but i'll do it on command line if necessary. or just kill this thing and make a new PR.

**Pull request merged**